{
  "error": "An exception occurred during backtesting: 'open_order_last_update'",
  "traceback": [
    "Traceback (most recent call last):\n  File \"/backend-api/routers/manage_backtesting.py\", line 87, in run_backtesting\n    backtesting_results = await backtesting_engine.run_backtesting(\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/backend-api/lib/python3.12/site-packages/hummingbot/strategy_v2/backtesting/backtesting_engine_base.py\", line 97, in run_backtesting\n    executors_info = await self.simulate_execution(trade_cost=trade_cost)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/backend-api/lib/python3.12/site-packages/hummingbot/strategy_v2/backtesting/backtesting_engine_base.py\", line 130, in simulate_execution\n    for action in self.controller.determine_executor_actions():\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/backend-api/bots/controllers/generic/pmm.py\", line 261, in determine_executor_actions\n    actions.extend(self.stop_actions_proposal())\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/backend-api/bots/controllers/generic/pmm.py\", line 347, in stop_actions_proposal\n    stop_actions.extend(self.executors_to_early_stop())\n                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/backend-api/bots/controllers/generic/pmm.py\", line 364, in executors_to_early_stop\n    executors_to_early_stop = self.filter_executors(\n                              ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/backend-api/lib/python3.12/site-packages/hummingbot/strategy_v2/controllers/controller_base.py\", line 186, in filter_executors\n    return [executor for executor in executors if filter_func(executor)]\n                                                  ^^^^^^^^^^^^^^^^^^^^^\n  File \"/backend-api/bots/controllers/generic/pmm.py\", line 366, in <lambda>\n    filter_func=lambda x: x.is_active and x.is_trading and self.market_data_provider.time() - x.custom_info[\"open_order_last_update\"] > self.config.cooldown_time)\n                                                                                              ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\nKeyError: 'open_order_last_update'\n"
  ]
}